ada_feeding/launch/ada_feeding_launch.xml

```diff
@@ -11,11 +11,13 @@
   <!-- Launch the watchdog -->
   <node pkg="ada_feeding" exec="ada_watchdog.py" name="ada_watchdog">
     <param from="$(find-pkg-share ada_feeding)/config/ada_watchdog.yaml"/>
+    <remap from="~/ft_topic" to="/wireless_ft/ftSensor1" />
   </node>
 
   <!-- Launch the action servers necessary to move the robot -->
   <node pkg="ada_feeding" exec="create_action_servers.py" name="ada_feeding_action_servers">
     <param from="$(find-pkg-share ada_feeding)/config/ada_feeding_action_servers.yaml"/>
+    <remap from="~/watchdog" to="/ada_watchdog/watchdog" />
```

Comment:
Ah, using relative topics (like you did here) is better than absolute topics ([like I did in SegmentFromPoint](https://github.com/personalrobotics/ada_feeding/blob/b251e15ecac9edd49f595d84d920b3d087a75b48/ada_feeding_perception/launch/ada_feeding_perception_launch.xml#L7)). If you want to fix it in this PR (in both the launchfile and the ROS node) you're welcome to, otherwise I'll do it.

Reply:
Coming to think of it, I think there needs to be a separate version of this PR for `ada_feeding_perception` (including to address linter warnings), so I can be in charge of that.